frontend/tailwind.config.js

```diff
@@ -168,17 +170,34 @@ export default {
       bigHeader: '3rem',
       biggerHeader: '3.125rem',
       biggestHeader: '4rem',
+      },
+      extend: {
+        maxWidth: {
+          '1440': '1440px',
+          '800': '800px',
+          '960': '960px'
+        },
+        maxHeight: {
+          '100': '100px',
+          '770': '770px'
+        }, 
+        borderRadius: {
+          '10p': '10%',
+          '30': '30px',
+          '40': '40px'
+  //     '4xl': '2rem',
+        }, 
+        minWidth: {
+          '200': '200px'
+        },
+        padding: {
+          '15': '15%',
+        },
```

Comment:
i think as we extend the tailwind config, we should aim to name the preset values by what they represent, or in relative terms, or what they are used for, rather than duplicating the value. 

We should probably keep the standard breakpoint sizes 
      sm: '600px',
      md: '960px',
      lg: '1280px',
      xl: '1920px',
      
      and then add others as needed, so if we really need 1440px (and using `lg` at 1280px isnt close enough, let's name it like `"blog-max-width": "1440px"` or something more like that.